easybuild/easyconfigs/d/destiny/destiny-2.5.6-intel-2017a-R-3.4.0.eb

```diff
@@ -0,0 +1,29 @@
+easyblock = 'RPackage'
+
+name = 'destiny'
+version = '2.5.6'
+versionsuffix = '-R-%(rver)s'
+
+homepage = 'https://bioconductor.org/packages/destiny'
+description = "R packages to create and plot diffusion maps."
+
+toolchain = {'name': 'intel', 'version': '2017a'}
+
+source_urls = [
+    'http://bioconductor.org/packages/release/bioc/src/contrib/',
+    'https://bioconductor.org/packages/devel/bioc/src/contrib/',
+]
+sources = ['%(name)s_%(version)s.tar.gz']
+checksums = ['134210f9b8d6d58309a1f886065175c723e2612b4be819beb9e9fe5707b0f2ac']
+
+dependencies = [
+    ('R', '3.4.0', '-X11-20170314'),
+    ('R-bundle-Bioconductor', '3.5', versionsuffix),
+]
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': [name],
+}
+
+moduleclass = 'data'
```

Comment:
`R packages to create and plot diffusion maps.`

To me, this fits `data` (or maybe `vis`) better than `bio`?